t/plugin/http-logger-log-format.t

```diff
@@ -438,7 +439,7 @@ done
                 [[{
                     "log_format": {
                         "host": "$host",
-                        "labels": "$a6_route_labels",
+                        "labels": "custom variable check",
```

Comment:
Why you change this?

Reply:
Actually the labels passed in route were not being reflected on the log so I had removed $a6_route_labels. Checking this right now...

Reply:
Okay this ctx variable was set here https://github.com/apache/apisix/blob/73ac77d22b4ae0c2b597105e778f9a5d6ea86e3d/t/plugin/http-logger-log-format.t#L491
Added it back